students/15th/turtle/hanbyul-kim/user/views.py

```diff
@@ -0,0 +1,100 @@
+
+import json, re
+import bcrypt
+import jwt
+
+from django.http        import JsonResponse
+from django.views       import View
+from django.db          import IntegrityError
+from django.db.models   import Q
+
+from user.models        import User
+
+
+
+class UserView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+
+        try:
+            data['name']    =   data.get('name')
+            data['email']   =   data.get('email')
+            data['mobile_number']   =   data.get('mobile_number')
+
+            if len(data['password']) < 8:
+                return JsonResponse({"message":"password must be at least 8 characters"},status=400)
+
+            email_reg='[a-zA-Z0-9_-]+@[a-z]+.[a-z]'
+            email_validation = re.compile(email_reg)
+
+            if 'email' in data:
+                if not re.match(email_validation, str(data.get('email'))):
+                    return JsonResponse({"message":"it is not a vaild address"},status = 400)
+
+            if User.objects.filter(Q(name=data['name']) & Q(email=data['email']) & Q(mobile_number=data['mobile_number'])).exists():
```

Comment:
`and` 조건에서는 Q객체를 사용하실 필요가 없습니다.
```suggestion
            if User.objects.filter(Q(name=data['name']) & Q(email=data['email']) & Q(mobile_number=data['mobile_number'])).exists():
```